blas/unit_test/Test_Blas1_axpby_unification.hpp

```diff
@@ -54,6 +54,12 @@
 #include <KokkosBlas1_axpby.hpp>
 #include <KokkosKernels_TestUtils.hpp>
 
+#if (KOKKOSKERNELS_DEBUG_LEVEL > 0)
```

Comment:
should this one also chekc > 1?

Reply:
The idea is that if you have `KOKKOSKERNELS_DEBUG_LEVEL=1` I will raise it to two for you so you get output from this local test. So I only need to check for `>0`

Reply:
Got it, I missed that this was in the unit test and not the blas src.